mbta.js

```diff
@@ -0,0 +1,34 @@
+const stopsBetweenStations = function(startLine, startStation, endLine, endStation){
+    var subwayLines ={ Red:['South Station', 
+                            'Park Street',
+                            'Kendall', 
+                            'Central',
+                            'Harvard',
+                            'Porter',
+                            'Davis',
+                            'Alewife'],
+                        Green:['Government Center',
+                            'Park Street',
+                            'Boylston',
+                            'Arlington',
+                            'Copley',
+                            'Hynes',
+                            'Kenmore'],
+                        Orange:['North Station',
+                                'Haymarket',
+                                'Park Street',
+                                'State',
+                                'Downtown Crossing',
+                                'Chinatown',
+                                'Back Bay',
+                                'Forest Hills'],};
+    
+    startStation = subwayLines[startLine].indexOf(startStation);
```

Comment:
Park Street is not always stop number 2, in the Orange line "Park Street" is number 3. And gives a wrong result.